addons/sales_team/sales_team.xml

```diff
@@ -128,6 +126,17 @@
                                     </kanban>
                                 </field>
                             </page>
+                            <page string="More Info">
```

Comment:
Ok, but I think it's redundant, as the dropdown on the Sales Team kanban view is restricted to the group sale manager.